seahorn/include/utils.h

```diff
@@ -14,8 +17,8 @@ extern void klee_assume(uintptr_t condition);
 
 #ifdef __KLEE__ // define macros for KLEE_ASSUME, the assumption for KLEE only
 #define KLEE_ASSUME(X) klee_assume(X)
-#else 
-#define KLEE_ASSUME(X) 
+#else
+#define KLEE_ASSUME(X) assume(X)
```

Comment:
this is really bad! This changes how SeaHorn interprets bencharks
`KLEE_ASSUME` are assumptions that are specific to KLEE.
If you want to use them in fuzzing, map `KLEE_ASSUME` to what you need but only if fuzzing is enabled!